instrumentation/github.com/aws/aws-sdk-go/service/helper/helper.go

```diff
@@ -0,0 +1,30 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
This package seems superfluous. The `AppendSpanAndTraceIDFromSpan` function can live unexported in the `otels3` package.